### PR TITLE
develop→master

### PR DESCRIPTION
**Pull request by exaexa**

### CODE REVIEW

**Comment by cylon-x** [comment]
:red_square: &nbsp;Commit 44d64482 requires formatting!

Required formatting changes summary:
```
```

**Review by stelmo — APPROVED** [review]
🚀

File: docs/make.jl [hunk 1]
```diff
@@ -2,16 +2,11 @@ using Documenter
...
-if "CI_EXTERNAL_PULL_REQUEST_IID" in keys(ENV)
-    ENV["TRAVIS_PULL_REQUEST"] = ENV["CI_EXTERNAL_PULL_REQUEST_IID"]
-else
-    ENV["TRAVIS_PULL_REQUEST"] = "false"
-end
+# some settings
+dev_docs_folder = "dev"
+pages_branch = "gh-pages"
+github_repo_slug = "LCSB-BioCore/COBREXA.jl"
+delete!(ENV, "GITHUB_REPOSITORY")
```
Comment (laurentheirendt) [inline docs/make.jl:9]: I think we should remove this env variable on Gitlab and not here - can you change?

File: docs/make.jl [hunk 2]
```diff
@@ -2,16 +2,11 @@ using Documenter
...
-# set the merge/pull request ID
-if "CI_EXTERNAL_PULL_REQUEST_IID" in keys(ENV)
-    ENV["TRAVIS_PULL_REQUEST"] = ENV["CI_EXTERNAL_PULL_REQUEST_IID"]
-else
-    ENV["TRAVIS_PULL_REQUEST"] = "false"
-end
+# some settings
+dev_docs_folder = "dev"
+pages_branch = "gh-pages"
+github_repo_slug = "LCSB-BioCore/COBREXA.jl"
```
Comment (laurentheirendt) [inline docs/make.jl:8]: this should be retrievable from the env

File: README.md [hunk 3]
```diff
@@ -18,7 +18,7 @@
 [cov-url]: https://codecov.io/gh/LCSB-BioCore/COBREXA.jl
 
 [contrib-img]: https://img.shields.io/badge/contributions-start%20here-green
-[contrib-url]: https://github.com/LCSB-BioCore/COBREXA.jl/blob/master/.github/CONTRIBUTING.md
+[contrib-url]: https://lcsb-biocore.github.io/COBREXA.jl/dev/howToContribute/
```
Comment (laurentheirendt) [inline README.md:21]: The URL should be `stable` here I think

Comment (exaexa) [reply README.md:21]: contribution kindof implies development so I pointed it to the (likely more relevant) dev version

File: docs/make.jl (hunk 1 above)
Comment (exaexa) [reply docs/make.jl:9]: I can't modify gitlab settings but you can safely erase this one there now. Then I can remove this from here

File: docs/make.jl (hunk 2 above)
Comment (exaexa) [reply docs/make.jl:8]: Doesn't seem so, I'd need to parse it out of EXTERNAL_REPO.

File: README.md (hunk 3 above)
Comment (laurentheirendt) [reply README.md:21]: contributing guide should be the one from `master`, i.e. the stable version. 

Although a developer might consult the development guide, it should be the stable version

File: docs/make.jl (hunk 2 above)
Comment (laurentheirendt) [reply docs/make.jl:8]: this script executes on gitlab CI, so we can use `CI_PROJECT_NAMESPACE` and `CI_PROJECT_NAME`

File: docs/make.jl (hunk 1 above)
Comment (laurentheirendt) [reply docs/make.jl:9]: we can remove it, but then let's use other variables (see comment below)

File: README.md (hunk 3 above)
Comment (exaexa) [reply README.md:21]: ok

File: docs/make.jl (hunk 2 above)
Comment (exaexa) [reply docs/make.jl:8]: but that doesn't match the github ones (that I need) right?

Comment (laurentheirendt) [reply docs/make.jl:8]: it is the same as the gitlab one, as it is a mirror

Comment (exaexa) [reply docs/make.jl:8]: ok, fixed in #500

File: docs/make.jl (hunk 1 above)
Comment (exaexa) [reply docs/make.jl:9]: Removed completely (see #500)

**Pull request merged**